tests/test_integration.py

```diff
@@ -818,8 +880,10 @@ def global_modeling_events_plus_regressors():  ### GLOBAL MODELLING + REGRESSORS
             for frst in forecast:
                 fig = m.PLOT(frst)
```

Comment:
why capital letters?

Reply:
I wonder the same haha I will change it.